config/wpcalypso.json

```diff
@@ -44,7 +44,6 @@
 		"domains/new-status-design/auto-renew": true,
 		"domains/new-status-design/new-options": true,
 		"domains/new-status-design/security-option": true,
-		"editor/before-deprecation": true,
```

Comment:
In general we usually mirror stage, wpcalypso and production values, unless there's a specific reason otherwise. 

Folks will get confused if stage and production diverge, and wpcalypso is used for E2E tests.